examples/mach_eval_examples/SynR_eval/SynR_flux_linkage_step.py

```diff
@@ -0,0 +1,72 @@
+import os
+import copy
+
+from mach_eval import AnalysisStep, ProblemDefinition
+from mach_eval.analyzers.electromagnetic import flux_linkage_analyzer as flux_linkage
+from mach_eval.analyzers.electromagnetic.flux_linkage_analyzer_config import Flux_Linkage_Config
+
+############################ Define Flux Linkage Step ###########################
+class SynR_EM_ProblemDefinition(ProblemDefinition):
+    """Converts a State into a problem"""
+
+    def __init__(self):
+        pass
+
+    def get_problem(state):
+
+        problem = flux_linkage.Flux_Linkage_Problem(
+            state.design.machine, state.design.settings)
+        return problem
+
+# initialize em analyzer class with FEA configuration
+configuration = Flux_Linkage_Config(
+    no_of_rev = 1,
+    no_of_steps = 72,
+
+    mesh_size=3, # mm
+    mesh_size_rotor=1.5, # mm
+    airgap_mesh_radial_div=4,
+    airgap_mesh_circum_div=720,
+    mesh_air_region_scale=1.05,
+
+    only_table_results=False,
+    csv_results=("FEMCoilFlux"),
+    del_results_after_calc=False,
+    run_folder=os.path.dirname(__file__) + "/run_data/",
+    jmag_csv_folder=os.path.dirname(__file__) + "/run_data/jmag_csv/",
+
+    max_nonlinear_iterations=50,
+    multiple_cpus=True,
+    num_cpus=4,
+    jmag_scheduler=False,
+    jmag_visible=True,
+    non_zero_end_ring_res = False,
+    scale_axial_length = True,
+    jmag_version = "21.1",
+    time_step = 0.0001
+)
+
+class SynR_Flux_Linkage_PostAnalyzer:
+    
+    def get_next_state(results, in_state):
+        state_out = copy.deepcopy(in_state)
+
+        state_out.conditions.path = results["csv_folder"]
+        state_out.conditions.study_name = results["study_name"]
+        state_out.conditions.I_hat = results["current_peak"]
+        state_out.conditions.rotor_angle = results["rotor_angle"]
+        state_out.conditions.name_of_phases = results["name_of_phases"]
+
+        print("\n************************ FLUX LINKAGE RESULTS ************************")
+        print("path = ", state_out.conditions.path)
+        print("study_name = ", state_out.conditions.study_name)
+        print("I_hat = ", state_out.conditions.I_hat, " A")
+        print("rotor_angle = ", state_out.conditions.rotor_angle, " deg")
```

Comment:
This is an optional change but when we print the rotor_angle array, it would be better if we have theese vaulues without mention of "array" itself. Could you consider if this change is acceptable for you?

```suggestion
        print("rotor_angle = ", state_out.conditions.rotor_angle[0], " deg")
```

I am expecting this result:
![image](https://github.com/Severson-Group/eMach/assets/114006024/e36e9374-6c85-451d-87c8-c5783a3b11b3)

Reply:
Thank you @noguchi-takahiro, I have made this change.